config/settings.yml

```diff
@@ -1,3 +1,4 @@
+control_master_host: sdr-infra
```

Comment:
I wonder if we should also toss a sentence or two at the README about this new CM stuff, including the config. But I'm approving either way. Thank you for working on this!